apps/web/layouts/PageLayout/AccountMenu.tsx

```diff
@@ -4,8 +4,8 @@ import {
   DropdownMenuItem,
   DropdownMenuTrigger,
 } from '@usevenice/ui'
-import {supabase} from 'contexts/common-contexts'
-import {useSession} from 'contexts/session-context'
```

Comment:
@bhongy this is something I didn't have time to look into yet - why does typescript automatically try to import like this rather than using the proper relative import path. Would be nice to look into if you get a chance

Reply:
@tonyxiao Sounds good; I will. The build works (next resolves the import) but I suspect there's something wrong in our tsconfig.